README.md

```diff
@@ -31,6 +28,8 @@ The component uses the [PyETo module to calculate the evapotranspiration value (
 
 This is all the component does, and this is on purpose to provide maximum flexibility. Users are expected to use the value of `sensor.[zone_name]` to interact with their irrigation system and afterwards call the `smart_irrigation.reset_bucket` service. [See the example automations below](#step-4-creating-automations).
 
+Smart Irrigation custom component for Home Assistant. Partly based on the excellent work at https://github.com/hhaim/hass/.
```

Comment:
I think we can remove this. None of his work is still present in v2. the BSI, HART, DART, WB stuff... all of it is gone.

Reply:
done